packages/framework/package.json

```diff
@@ -104,6 +104,7 @@
     "zod-to-json-schema": "^3.23.0"
   },
   "dependencies": {
+    "@novu/shared": "workspace:*",
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** move this one to `devDependencies`‏

Reply:
We will keep it like this for now as decided with Richard, and revisit it later.